web/assets/main.scss

```diff
@@ -63,11 +63,6 @@ $govuk-assets-path: "/lpa/frontend/assets/";
     width: fit-content;
 }
 
-.app-\!-search-results-table th,
-.app-\!-search-results-table th button {
-    color: govuk-colour("blue");
```

Comment:
since we're temporarily removing the sorting functionality on the headings, I have removed this so that the headings are now black, otherwise users could get confused thinking they are a clickable link (verified by claire)